### DISCUSSION TITLE
v3最新版上传文件显示Network Error

### DISCUSSION THREAD

**Discussion by Startear187** [AlistGo/alist#2016] [Q&A]
v3.2版本上传文件会出现Network Error，之前v2版本就没有。
![1](https://user-images.githubusercontent.com/63627982/195847282-2059510e-be8c-4573-a36c-16711685d0bd.png)

**Comment by PastWind-Zhou** [comment]
这个问题解决了吗？我也遇到了

**Reply by Startear187** [reply]
不知道，重新传吧，把stream换另一个试试

**Reply by PastWind-Zhou** [reply]
我这stream直接就不行，换了另一个就报NetworkError

**Reply by Zayia** [reply]
问题解决了吗
